src/protections/WordList.ts

```diff
@@ -89,19 +83,25 @@ export class WordList extends Protection {
                     return
                 }
             }
-
+            if (!Boolean(this.badWords)) {
+                // Create a mega-regex from all the tiny baby regexs
+                this.badWords = new RegExp(
+                    "(" + mjolnir.config.protections.wordlist.words.join(")|(") + ")",
```

Comment:
Not your fault, but that's rather ugly and insecure. We'll need to fix this at some point.

Reply:
In case of error during regexp compilation, will this be logged somewhere? What will break?

Reply:
Not sure what to do with it tbh, i've wrapped it so we can tell it fails but it doesn't feel good